pyslackersweb/website/views.py

```diff
@@ -59,6 +60,12 @@ class SlackView(web.View):
             "errors": {},
         }
 
+    def allowed_email(self, email: str) -> bool:
+        _, domain = email.split("@")
```

Comment:
Let's check that `@` is in the email before splitting, just in case someone submits an incomplete email.